tests/benchmark/tests/PyRosetta.py

```diff
@@ -12,6 +12,9 @@
 ## @brief  PyRosetta binding self tests
 ## @author Sergey Lyskov
 
+# A bit of Python magic here, what we trying to say is this: from ../__init__ import *, but init path is calculated relatively to this location
+exec(open('/'.join(__file__.split('/')[:-1]) +  '/__init__.py').read(), globals())
+
```

Comment:
@roccomoretti i think this will be the exact replacement if we want to avoid using direct `exec`:

```Python
from importlib.machinery import SourceFileLoader
SourceFileLoader(__name__, '/'.join(__file__.split('/')[:-1]) +  '/__init__.py').load_module()
```